### PR TITLE
lambda fn for ngrams added; auth in request is also enabled;

### PR DESCRIPTION
**Pull request by ranjan-stha**
* Ngrams lambda func. added
* Support for French, Spanish, English.
* Auth is integrated

### CODE REVIEW

File: src/ngrams/handler.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    "enable_stopwords": false,
+    "enable_stemming": true,
+    "enable_case_sensitive": false
+    "max_ngrams_items": 10
+}
+"""
+
+MAX_NGRAMS_ITEMS = 10
+
+stemmer_en = nltk.stem.PorterStemmer()
```
Comment (k9845) [inline src/ngrams/handler.py]: Just curious..why aren't we using `EnglishStemmer()` here?

File: src/ngrams/handler.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+stemmer_fr = FrenchStemmer()
+stemmer_es = SpanishStemmer()
+
+stopwords_en = stopwords.words('english')
+stopwords_fr = stopwords.words('french')
+stopwords_es = stopwords.words('spanish')
```
Comment (k9845) [inline src/ngrams/handler.py:37]: Why are we providing stopwords besides these 3 languages?

File: src/ngrams/handler.py (hunk 1 above)
Comment (ranjan-stha) [reply src/ngrams/handler.py]: @k9845 Updated to EnglishStemmer from snowball.

File: src/ngrams/handler.py (hunk 2 above)
Comment (ranjan-stha) [reply src/ngrams/handler.py:37]: @k9845 The entries primarily are from these three languages (as discussed with Jean)

**Pull request merged**